robot/main.py

```diff
@@ -17,4 +17,27 @@
 
 
 # Write your program here.
-ev3.speaker.beep()
+#ev3.speaker.beep()
+left_motor = Motor(Port.A)
+right_motor = Motor(Port.B)
+obstacle_sensor = UltrasonicSensor(Port.S4)
+
+#motor.run_time(360, 10000, then=Stop.HOLD, wait=True)
+
+robot = DriveBase(left_motor, right_motor, wheel_diameter=50, axle_track=90)
+
+while True:
```

Comment:
Put this whole while loop into a test function and then run it in "if __name__ == 'main':"